docs/services/web_power_switch.rst

```diff
@@ -1,14 +1,47 @@
 Web Power Switch
 ================
 
+This service controls a power switch that is controllable over the internet.
+
+So far catkit2 has been tested on the LPC7-PRO web power switch from TeleDynamics:
+https://www.teledynamics.com/#/productdetails/LPC7-PRO
```

Comment:
This is minor, but could you reformat this into a bullet list like you did for the ZWO camera models? This is also how we have it in some other service docs and makes it simpler to extend later on (and to find where to extend).